### PR TITLE
Travis test build ovs fix

### PR DESCRIPTION
**Pull request by kiran1048**

### CODE REVIEW

File: test/travis_test/build_2_container.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+docker build -f $code_dir/build/Dockerfile -t aca_build0 .
+echo "    --- images list ---"
+docker images
+echo "    --- container list ---"
+docker ps -a
+echo "    --- create container ---"
+docker rm -f aca_PARENT || true
+docker create -v $code_dir:/mnt/host/code -it --privileged --cap-add=NET_ADMIN --cap-add=SYS_PTRACE --security-opt seccomp=unconfined --name aca_PARENT aca_build0:latest /bin/bash
+docker start aca_PARENT
+docker exec aca_PARENT bash -c "/etc/init.d/openvswitch-swtich restart && \
```
Comment (er1cthe0ne) [inline test/travis_test/build_2_container.sh:25]: typo in openvswitch-swtich

File: test/travis_test/build_2_container.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+ovs-vswitchd --pidfile --detach"
+docker rm -f aca_CHILD || true
+docker create -v $code_dir:/mnt/host/code -it --privileged --cap-add=NET_ADMIN --cap-add=SYS_PTRACE --security-opt seccomp=unconfined --name aca_CHILD aca_build0:latest /bin/bash
+docker start aca_CHILD
+docker exec aca_CHILD bash -c "/etc/init.d/openvswitch-swtich restart && \
```
Comment (er1cthe0ne) [inline test/travis_test/build_2_container.sh:30]: typo in openvswitch-swtich

File: test/travis_test/build_2_container.sh [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+docker exec aca_PARENT bash -c "/etc/init.d/openvswitch-switch restart && \
+ovs-vswitchd --pidfile --detach"
+docker rm -f aca_CHILD || true
+docker create -v $code_dir:/mnt/host/code -it --privileged --cap-add=NET_ADMIN --cap-add=SYS_PTRACE --security-opt seccomp=unconfined --name aca_CHILD aca_build0:latest /bin/bash
+docker start aca_CHILD
+docker exec aca_CHILD bash -c "/etc/init.d/openvswitch-switch restart && \
+ovs-vswitchd --pidfile --detach"
+echo "    --- container list ---"
+docker ps -a
+exit
```
Comment (er1cthe0ne) [inline test/travis_test/build_2_container.sh:34]: I suspect this "exit" is causing the script to exit here without running the tests below

**Review by er1cthe0ne — APPROVED** [review]
Looks good. Thanks for completing the Travis CI setup and addressing the issue flagged in the unit test.

**Comment by kiran1048** [comment]
Thank you Eric.

**Pull request merged**